### ISSUE TITLE
Restart workloads after Istio uninstallation

### ISSUE THREAD

**Issue by strekm** [kyma-project/istio#107]
**Description**

After Istio is uninstalled all "remainings" should also be removed from a cluster. Istio uninstallation leave proxies on workloads. Implement functionality in "proxy reset" that after Istio uninstallation will remove proxies from workloads. 

ACs:
- [x] there's no proxies on workloads after Istio uninstallation

**Reasons**

Neat clean up after Istio uninstallation

**DoD:**
~~- [ ] provide documentation~~
- [x] provide unit tests
- [x] test on Gardener
~~- [ ] verify resource limits~~
~~- [ ] followup issue~~
~~- [ ] publish release notes and _What's New_ updates for Kyma customers~~

**Attachments**
depends on: https://github.com/kyma-project/istio/issues/81
part of: https://github.com/kyma-project/istio/issues/38

**PR**
- https://github.com/kyma-project/istio/pull/111
